tests/test_main.py

```diff
@@ -76,9 +76,10 @@ def test_interpolation_mutatormath(data_dir, tmp_path):
 
 
 def test_interpolation_mutatormath_source_layer(data_dir, tmp_path):
-    shutil.copyfile(
-        data_dir / "MutatorSans" / "MutatorSans.designspace",
-        tmp_path / "MutatorSans.designspace",
+    shutil.copytree(
+        data_dir / "MutatorSans",
```

Comment:
why did you need to copy the sources too?

Reply:
Because otherwise this test will fail out because of the compatibility checker not being able to find the sources; we need to get past that in order to test for the "layer in designspace" thing we're testing for in this particular test.